tools/openshift-ci/Dockerfile

```diff
@@ -3,34 +3,26 @@ FROM quay.io/centos/centos:7
 LABEL vendor="Red Hat inc."
 LABEL maintainer="OCP QE Team"
 
-WORKDIR /root
 USER root
 
+ADD . /verification-tests/
+
+RUN set -x && \
+    mv /tierN/ /verification-tests/tierN/
+
 RUN set -x && \
     yum -y update && \
     SCL_BASE_PKGS="centos-release-scl scl-utils-build" && \
-    INSTALL_PKGS="rh-ruby27 rh-ruby27-ruby-devel bsdtar git openssh-clients" && \
+    INSTALL_PKGS="rh-ruby26 rh-ruby26-ruby-devel rh-git218 bsdtar" && \
```

Comment:
Perhaps use ruby27 ? as the image we currently use is using ruby27.

Reply:
Thanks Liang, fixed that